rotkehlchen/db/drivers/db_messages.proto

```diff
@@ -0,0 +1,56 @@
+syntax = "proto3";
+
+enum DBConnectionType {
+  USER = 0;
+  TRANSIENT = 1;
+  GLOBAL = 2;
+}
+
+enum DBMethod {
+  INITIALIZE = 0;
+  OPEN_CURSOR = 1;
+  CLOSE_CURSOR = 2;
+  EXECUTE = 3;
+  EXECUTEMANY = 4;
+  EXECUTESCRIPT = 5;
+  ROLLBACK = 6;
+  COMMIT = 7;
+  CLOSE = 8;
+}
+
+message DBWriteResult {
+  int32 rowcount = 1;
+  uint64 lastrowid = 2;
+}
+
+message DBWriterError {
+  string message = 1;
+  optional string module = 2;
+  string class_name = 3;
+}
+
+message TypedArray {
+  repeated TypedData array = 1;
+}
+
+message TypedData {
+  string type = 1;
+  oneof data {
+    bytes bytes = 2;
+    TypedArray array = 3;
+  }
+}
```

Comment:
add a comment saying that typedArray is needed because `oneof doesn't allow repeated as label for their members`

Reply:
https://stackoverflow.com/a/62239063